pkg/dao/rpms.go

```diff
@@ -56,7 +57,10 @@ func (r rpmDaoImpl) List(orgID string, repositoryConfigUUID string, limit int, o
 		}
 		return api.RepositoryRpmCollectionResponse{},
 			totalRpms,
-			fmt.Errorf("repositoryConfigUUID = %s is not owned", repositoryConfigUUID)
+			&ce.DaoError{
+				NotFound: true,
+				Message:  fmt.Sprintf("repositoryConfigUUID = %s is not owned", repositoryConfigUUID),
```

Comment:
I think it could make sense to change the error message to "Could not find repository with UUID..." to make this more consistent with other endpoints. What do you think?

Reply:
Yeah that definitely makes sense.